hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/client/api/impl/TimelineConnector.java

```diff
@@ -145,14 +145,14 @@ protected void serviceInit(Configuration conf) throws Exception {
         @Override
         public HttpURLConnection configure(HttpURLConnection conn)
             throws IOException {
-          setTimeouts(conn, DEFAULT_SOCKET_TIMEOUT);
+          setTimeouts(conn, 60_000);
```

Comment:
Nit: the behaviour changed a bet, if someone simply edits the socketTimeout variable, this won't be changed with it. Shouldn't the socketTimeout be used here as well?

Reply:
@brumi1024 Thanks for reviewing the code, I will improve it.